components/molecule/photoUploader/README.md

```diff
@@ -0,0 +1,23 @@
+# MoleculePhotoUploader
+
+> Description
```

Comment:
Maybe some description with the amazing features that it supports will be nice! ;)

Reply:
Added! Also, PropTypes info improved, and the description of this `PR`.